cluster/saltbase/salt/kube-proxy/init.sls

```diff
@@ -17,7 +17,12 @@
     - makedirs: true
     - dir_mode: 755
     - context:
-        cpurequest: '200m'
+    # 20m might cause kube-proxy CPU starvation on full nodes, resulting in
```

Comment:
nit: please indent with cpurequest (I wouldn't bring this nit up if I didn't already have a reason to re-push, which is the other comment)